.github/workflows/test.yml

```diff
@@ -10,6 +10,18 @@ env:
   BUNDLE_WITHOUT: release
 
 jobs:
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install Ruby ${{ matrix.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.1"
```

Comment:
why only 3.1 here?

Reply:
we have a dependency to a vmware lib, which doesn't support ruby 3.2 yet :(